apps/utils/keystone.py

```diff
@@ -53,15 +54,16 @@ def get_researchgroup_id(keystone_url: str, account_name: str, auth_header: dict
 
 
 def get_earliest_startdate(alloc_requests: [dict]) -> date:
-    """Given a number of requests, determine the earliest start date across them"""
+    """Given a number of requests, determine the earliest start date across them. This takes the most recent rawusage
```

Comment:
Minor styling note.

Our team tends to use the Google style guide, which is an extension of PEP8. The first line of docstring should fit on a single line. If you need to add extra details, add a white space followed by a longer multi-line clarification.

see here: https://android.googlesource.com/platform/external/google-styleguide/+/refs/tags/android-s-beta-2/pyguide.md#3_8-comments-and-docstrings

Reply:
Thanks for the heads up, my bad for not just getting a quick review on this one. Is there an easy way to get pycharm to point this kind of thing out to me?